contrib/demo/.startUtils

```diff
@@ -0,0 +1,35 @@
+wait_command() {
+  local command="$1"; shift
+  local wait_seconds="${1:-40}"; shift # 40 seconds as default timeout
+
+  until [[ $((wait_seconds--)) -eq 0 ]] || eval "$command &> /dev/null" ; do sleep 1; done
+
+  ((++wait_seconds))
+}
+
+wait_file() {
```

Comment:
`wait_file` appears to be unused in this change. Can we delete it?

Reply:
I assume we can delete it, yes.

Reply:
Done